rear_camera_py/Makefile

```
@@ -10,6 +10,12 @@ help:
 	@echo "スタイルチェックを行う"
 	@echo " $$ make check_style"
 
+test:
+	python -m unittest
+
+run:
+	python -m rear_camera --debug
```

このコマンドは、デバック用のものですか？それとも、実行に必要？
もしデバック用なら
```suggestion
debug:
	python -m rear_camera --debug
```
にして欲しいです。